src/ponyc/main.c

```diff
@@ -81,6 +81,16 @@ int main(int argc, char* argv[])
   opt.ast_print_width = get_width();
   opt.argv0 = argv[0];
 
+  size_t args_size = 0;
+  for (int i = 1; i < argc; i++)
+    args_size += strlen(argv[i]) + 1;
+  opt.all_args = (const char*)calloc(args_size, sizeof(const char));
+  for (int i = 1; i < argc; i++)
+  {
+    strncat((char*)opt.all_args, argv[i], strlen(argv[i]));
+    strncat((char*)opt.all_args, " ", 1);
+  }
```

Comment:
what's the purpose of this?

Reply:
IIRC one of the LLVM 11 changes likes to see the argument string; I could be mistaken.

Reply:
Can we get a comment explaining this. I know it made me do a double take and then I still had no idea.

Reply:
Added; it's for LLVM debug info.